index.html

```diff
@@ -58,143 +60,7 @@ <h3 class="connect">LET’S CONNECT</h3>
-                            <img src="./Image/Counter.png" alt="circle-icon">
-                            <span class="num-text must-ignor">Back End Dv</span>
-                            <span class="num-text must-ignor2">Full Stack Dev</span>
-                            <img src="./Image/Counter.png" alt="circle-icon">
-                            <span class="num-text">2015</span>
-                        </div>
-                    </div>
-
-                    <p class="works-container-text">A daily selection of privately <br class="must-ignor">
-                        personalized reads;no accounts or<br class="must-ignor">
-                        sign-ups required.</p>
-
-                    <ul class="primarytext3">
-                        <li class="lang">HTML</li>
-                        <li class="lang must-ignor2">Ruby on Rails</li>
-                        <li class="lang">CSS</li>
-                        <li class="lang JavaScript">JavaScript</li>
-                    </ul>
-
-                    <div class="primarytext4">
-                        <button class="project">See project</button>
-                    </div>
-                </div>
-            </div>
-            <div class="works-container fist-works-container">
-                <img src="./Image/Snapshoot Portfolio (2).png" alt="gymfit-image" class="works-container-image must-ignor1">
-                <img src="./Image/desktop2.png" alt="availability-image" class="works-container-image must-ignor2">
-
-                <div class="wapper-90">
-                    <div class="primarytext1">
-                        <h2 class="primarytext-title must-ignor">Tonic</h2>
-                        <h2 class="primarytext-title must-ignor2">Facebook 360</h2>
-                        <div class="frame"> <span class="canopy must-ignor">CANOPY</span>
-                            <span class="canopy must-ignor2">FACEBOOK</span>
-                            <img src="./Image/Counter.png" alt="circle-icon">
-                            <span class="num-text must-ignor">Back End Dv</span>
-                            <span class="num-text must-ignor2">Full Stack Dev</span>
-                            <img src="./Image/Counter.png" alt="circle-icon">
-                            <span class="num-text">2015</span>
-                        </div>
-                    </div>
-
-                    <p class="works-container-text">A daily selection of privately <br class="must-ignor">
-                        personalized reads;no accounts or<br class="must-ignor">
-                        sign-ups required.</p>
-
-                    <ul class="primarytext3">
-                        <li class="lang">HTML</li>
-                        <li class="lang must-ignor2">Ruby on rails</li>
-                        <li class="lang">CSS</li>
-                        <li class="lang JavaScript">JavaScript</li>
-                    </ul>
```

Comment:
- [ ] Good work on your design but there are some aspects you can improve on. 
  - Your image's aspect ratio does not appear to be balanced. Kindly adjust the image's height and if the height is too long you can use a scroll bar.
  - Kindly position the close button at the top right corner to match the original design
  - Kindly also position the `tonic` header  correctly to match the original design

![image](https://user-images.githubusercontent.com/98453979/233112893-0b4f4e28-4cd1-4740-b736-2954f8fab6ef.png)
****

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/233113443-eb6efd9a-aa19-471b-92b9-0300a3263971.png)

Reply:
done